src/js/components/RangeInput/stories/Test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Actually can we remove this story? It is helpful for testing but I don't think we need to add it to storybook

Reply:
done!